### ISSUE TITLE
Prettier configs override prettierrc

### ISSUE THREAD

**Issue by texastoland** [Shopify/eslint-plugin-shopify#118]
`plugin:shopify/typescript-prettier` and `plugin:shopify/prettier` pass Prettier config inline overriding any external config. It can be worked around but it took me a few hours to figure out why:

```yaml
  prettier/prettier:
    - error
    - parser: typescript
```

**Comment by BPScott** [comment]
I'm fairly sure we can remove these overrides as we should trust the project's settings.

**Comment by texastoland** [comment]
I just [fixed typescript-eslint-parser](https://github.com/eslint/typescript-eslint-parser/pull/499) for TypeScript 3. I'll PR this and bump that at the same time.
